libs/protocol-client/src/lib/graphql/queries.graphql

```diff
@@ -451,6 +451,16 @@ query getDaoContributionCount(
   result: getDaoContributionCount(where: $where) 
 }
 
+query getDaoContributionCountByUser(
+  $where: GetContributionInput!
+) {
+  result: getDaoContributionCountByUser(where: $where) {
+    count
+    user_name 
```

Comment:
Nice to have `user_name` included on the return values -- perfect for the pie chart 🔥